src/frontend/components/TimedTextCreationForm/TimedTextCreationForm.tsx

```diff
@@ -103,11 +106,8 @@ export class TimedTextCreationForm extends React.Component<
     }
   }
 
-  onSelectChange(
-    option: Maybe<Nullable<ValueType<SelectOption>>>,
```

Comment:
Is there a reason we're sure it's not null or undefined?

Reply:
```
export type ValueType<OptionType> = OptionType | OptionsType<OptionType> | null | undefined;
```

so `Nullable` and `Maybe` are useless, no ?

Reply:
Indeed they are, thanks